src/ui/public/agg_types/buckets/migrate/include_exclude.js

```diff
@@ -0,0 +1,15 @@
+import _ from 'lodash';
+
+const migrateIncludeExcludeFormat = {
+  serialize: function (value) {
+    if (!value || _.isString(value)) return value;
+    else return value.pattern;
+  },
+  write: function (aggConfig, output) {
+    const value = aggConfig.params[this.name];
+    if (!value || _.isString(value)) output.params[this.name] = value;
+    else output.params[this.name] = value.pattern;
```

Comment:
if you were avoiding the {} brackets so this can be more succinct, consider using the ternary if-then-else,

`output.params[this.name] = (!value || _.isString(value) ? value : value.pattern;`